### PR TITLE
Aeotech Multisensor:added configuration get command

### PR DESCRIPTION
**Pull request by sjl2178**
Edited added handler to include a Configuration:Get() call to query for power source in Aeotech Multisensor 6 & 7

### CODE REVIEW

**Comment by sjl2178** [comment]
@greens Was this what you meant by updating init to call update_preference and then the Get call?

File: drivers/SmartThings/zwave-sensor/src/aeotec-multisensor/multisensor-6/init.lua [hunk 1]
```diff
@@ -18,8 +18,20 @@ local cc = require "st.zwave.CommandClass"
...
 
+local function device_added(self, device)
+  device:send(Configuration:Get({parameter_number = 9})) 
+  device:refresh()
+end
+
+local function device_init(self, device)
+  device:set_update_preferences_fn(preferences.update_preferences)
+  device:send(Configuration:Get({parameter_number = 9}))
+end
```
Comment (greens) [inline drivers/SmartThings/zwave-sensor/src/aeotec-multisensor/multisensor-6/init.lua:33]: added is correct, but here what I would do is set the update preferences function to a new function that is a composition of the standard preferences update function + the sending of the configuration get

**Review by greens — APPROVED** [review]
Great. The one thing I would change is to pull out the specific preference number (10 or 9) to and define it at the top of the file, since we're re-using it three times now.

**Comment by greens** [comment]
If you wanted to be super hardcore, you could probably use a device field set in the init function of the driver to define the specific preference we care about, and then refactor these two sub-drivers to be nested, but that's a lot of work for not much gain.

**Comment by sjl2178** [comment]
> If you wanted to be super hardcore, you could probably use a device field set in the init function of the driver to define the specific preference we care about, and then refactor these two sub-drivers to be nested, but that's a lot of work for not much gain.

I thought about that, but then figured that was a lot of work for a relatively straightforward fix lol

**Pull request merged**